modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -81,8 +202,43 @@ $ oc get network.config/cluster -o jsonpath='{.status.networkType}{"\n"}'
 ----
 $ oc get nodes
 ----
+
+.. If a node is stuck in the `NotReady` state, investigate the `machine-config-daemon-<seq>` pod logs and resolve any errors. The `<seq>` value is a random five character alphanumeric sequence.
+
+... To list the pods, enter the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-machine-config-operator
+----
 +
-If a node is stuck in the `NotReady` state, reboot the node again.
+.Example output
+[source,terminal]
+----
+NAME                                         READY   STATUS    RESTARTS   AGE
+machine-config-controller-75f756f89d-sjp8b   1/1     Running   0          37m
+machine-config-daemon-5cf4b                  2/2     Running   0          43h
+machine-config-daemon-7wzcd                  2/2     Running   0          43h
+machine-config-daemon-fc946                  2/2     Running   0          43h
+machine-config-daemon-g2v28                  2/2     Running   0          43h
+machine-config-daemon-gcl4f                  2/2     Running   0          43h
+machine-config-daemon-l5tnv                  2/2     Running   0          43h
+machine-config-operator-79d9c55d5-hth92      1/1     Running   0          37m
+machine-config-server-bsc8h                  1/1     Running   0          43h
+machine-config-server-hklrm                  1/1     Running   0          43h
+machine-config-server-k9rtx                  1/1     Running   0          43h
+----
+
+... To display the pod log for each machine config daemon pod shown in the previous output, enter the following command:
```

Comment:
I don't think it's quite clear enough that you need to repeat these two steps for each pod.
```suggestion
... Display the pod log for the first machine config daemon pod shown in the previous output by enter the following command:
```
Then add a step after line 241 that says something like "Repeat the previous two steps for each machine config daemon pod."